tests/FSharp.MongoDB.Bson.Tests/app.config

```diff
@@ -4,7 +4,7 @@
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
         <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-4.3.0.0" newVersion="4.3.0.0" />
+        <bindingRedirect oldVersion="0.0.0.0-999.999.999.999" newVersion="4.4.0.0" />
```

Comment:
I don't see why it would be necessary to force the tests to run with a particular version of FSharp.Core. The FSharp.MongoDB project is written to be compatible with F# 3.0 (i.e. FSharp.Core version 4.3.0.0). Having a version of FSharp.Core newer than 4.3.0.0 on your system should still be fine because version 4.3.0.0 of FSharp.Core is binary compatible with versions 4.3.1.0 and 4.4.0.0.

Could you provide me with logs/details on the "missing method exception" and information about your environment (e.g. operating system, F# version)?